content/en/platform/corda/5.0/application-networks/creating/mgm/register.md

```diff
@@ -152,4 +153,23 @@ Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -
 ```
 {{% /tab %}}
 {{< /tabs >}}
-If successful, you should see the `APPROVED` registration status.
+
+The returned status of the registration request will be one of the following options:
+
+* `APPROVED`: The registration request passed all validations and was approved by the MGM.
+* `NEW`: The member’s Corda cluster has accepted and persisted the registration request submitted via the REST API but is yet to process it.
+* `INVALID`: The member’s Corda cluster processed the registration request submitted via the REST API and determined the
+input to be invalid and did not attempt to send the request to join the network to the MGM’s Corda cluster.
+* `SENT_TO_MGM`: The member’s Corda cluster processed the registration request submitted via the REST API and found the input
+to be valid so the request to join the network was forwarded to the MGM’s Corda cluster.
+* `RECEIVED_BY_MGM`: The MGM has received a registration request that a member sent.
+* `DECLINED`: The MGM has received the registration request and rejected the request either automatically because of
+failed validation or manually by the Network Operator.
+* `PENDING_AUTO_APPROVAL`: The MGM has received the registration request and all automated validations were successful.
+The only remaining step is for the MGM to complete an automated approval before the member’s registration request takes effect.
+* `PENDING_MANUAL_APPROVAL`: The MGM has received the registration request and all automated validations were successful.
+The only remaining step is for the Network Operator to review and approve the registration via the MGM administration
+REST API before the member's registration request takes effect.
```

Comment:
Cross-ref to https://docs.r3.com/en/platform/corda/5.0/application-networks/managing/registration-requests/reviewing-registration-requests.html#reviewing-registration-requests?

Reply:
added